clients/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
I feel strongly that the Scala client should live in another repository and not in the main lakeFS one.

Reply:
I somewhat disagree: I have found it difficult to see the advantages of multi-repo.  But in any case a prerequisite for going multi-repo on our clients is to be able to publish our protobuf files to _somewhere_.  Saying "git submodules" does not tend to make friends...